src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/RandomNumberGenerator.cs

```diff
@@ -163,6 +164,200 @@ public static byte[] GetBytes(int count)
             return ret;
         }
 
+        /// <summary>
+        ///   Fills the elements of a specified span with items chosen at random from choices.
```

Comment:
Presumably this should match the wording on System.Random...

```suggestion
        ///   Fills the elements of a specified span with items chosen at random from the provided set of choices.
```